scripts/just.config.js

```diff
@@ -45,6 +46,7 @@ module.exports = function preset() {
   task('jest', jest);
   task('jest-watch', jestWatch);
   task('sass', sass);
+  task('ts:postprocess', postprocessTask);
```

Comment:
I almost feel like the other ones should be called `ts:postprocess:amd` because they are specialized. This one cuts across all lib types, which is why I named it `ts:postprocess`. It doesn't have to be `dts` specific, it just cuts across all output types.